test/runtime/test_vqeprogram.py

```diff
@@ -36,25 +39,42 @@ def setUp(self):
         super().setUp()
         self.provider = FakeRuntimeProvider()
 
-    @data({"name": "SPSA", "maxiter": 100}, SPSA(maxiter=100))
-    def test_standard_case(self, optimizer):
-        """Test a standard use case."""
+    def get_standard_program(self):
         circuit = RealAmplitudes(3)
         operator = Z ^ I ^ Z
         initial_point = np.random.random(circuit.num_parameters)
         backend = QasmSimulatorPy()
 
         vqe = VQEProgram(
             ansatz=circuit,
-            optimizer=optimizer,
+            optimizer=SPSA(),
             initial_point=initial_point,
             backend=backend,
             provider=self.provider,
         )
+        return vqe, operator
+
+    @data({"name": "SPSA", "maxiter": 100}, SPSA(maxiter=100))
+    def test_standard_case(self, optimizer):
+        """Test a standard use case."""
+        vqe, operator = self.get_standard_program()
+        vqe.optimizer = optimizer
         result = vqe.compute_minimum_eigenvalue(operator)
 
         self.assertIsInstance(result, VQEResult)
 
+    def test_supports_aux_ops(self):
+        """Test the VQEProgram says it supports aux operators."""
+        vqe, _ = self.get_standard_program()
+        self.assertTrue(vqe.supports_aux_operators)
```

Comment:
Do you also want to actually test the aux_operators somewhere?

Reply:
That would require running the VQE on runtime which we didn't include in the nature/optimization tests. We have tests in the runtime repo however, that do test this 🙂

Reply:
Running some tests in nature would be good generally, but that requires some more planning (maybe nightly runs would be better since it runs on the IBM simulator on the server) and provider access etc.

Reply:
Alright :+1: